src/Microsoft.Identity.Client/ApiConfig/GetAuthorizationRequestUrlParameterBuilder.cs

```diff
@@ -89,13 +123,19 @@ public new Task<Uri> ExecuteAsync(CancellationToken cancellationToken)
 
             if (ConfidentialClientApplication is IConfidentialClientApplicationExecutor executor)
             {
-                Validate();
-                return executor.ExecuteAsync((IGetAuthorizationRequestUrlParameters)Parameters, cancellationToken);
+                ValidateAndCalculateApiId();
+                return executor.ExecuteAsync(CommonParameters, Parameters, cancellationToken);
             }
 
             throw new InvalidOperationException(
                 "ConfidentialClientApplication implementation does not implement IConfidentialClientApplicationExecutor.");
         }
+
+        /// <inheritdoc />
+        internal override ApiEvent.ApiIds CalculateApiEventId()
+        {
+            return ApiEvent.ApiIds.None;
```

Comment:
what about adding an ID for this?

Reply:
I've added a work item for this.

---
In reply to: [249084553](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/798#discussion_r249084553) [](ancestors = 249084553)